doc/api/util.md

```diff
@@ -3102,6 +3102,23 @@ types.isExternal(new String('foo')); // returns false
 For further information on `napi_create_external`, refer to
 [`napi_create_external()`][].
 
+### `util.types.isFloat16Array(value)`
```

Comment:
This can be done fully on user land , why are we exposing it?

Reply:
Wouldn't it be inconsistent to not expose the 16 bit version next to all others?

Reply:
In the scope of this PR, the aim is on having this particular function on `node:internal/util/types`. [It is re-exported as `node:util/types` as is](https://github.com/nodejs/node/blob/b2405e9b075e5637f250cf57e93c0bd6f3189f9b/lib/util/types.js).

Reply:
It can't be done in userland easily, though, because you'd have to extract the Symbol.toStringTag getter. Consistency is a pretty strong justification for extending any API surface, on its own merits.